e2e/cypress/integration/notifications/desktop_notifications_spec.js

```diff
@@ -1,19 +1,10 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-// ***************************************************************
-// - [#] indicates a test step (e.g. # Go to a page)
-// - [*] indicates an assertion (e.g. * Check the title)
-// - Use element ID when selecting an element. Create one if none.
-// ***************************************************************
-
-// Stage: @prod
-// Group: @notifications
```

Comment:
Might have been unintentional but please revert and only remove the `// Stage: @prod`.